docs/components/movement-sensor/mpu6050.md

```diff
@@ -113,8 +95,7 @@ Then remove and fill in the attributes as applicable to your movement sensor, ac
 <!-- prettier-ignore -->
 | Name                  | Type    | Inclusion    | Description |
 | --------------------- | ------- | ------------ | ----------- |
-| `board`               | string  | **Required** | The `name` of the [board](/components/board/) to which the device is wired. |
-| `i2c_bus`             | string  | **Required** | The `name` of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on your [board](/components/board/) wired to this device. |
+| `i2c_bus`             | string  | **Required** | The index of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on your [board](/components/board/) wired to this device. |
```

Comment:
It's no longer configured on your board. I'd rephrase this part some more. I liked the phrasing in https://github.com/viamrobotics/docs/pull/2297/files: "The index of the [I2C bus] on the board that the sensor is wired to."

Reply:
:exploding_head: This file uses `I<sup>2</sup>C`, while the one in https://github.com/viamrobotics/docs/pull/2297 just uses I2C! I don't know which one we prefer, but I think they should be consistent from one to the other.